src/index.ts

```diff
@@ -1,71 +1,10 @@
 import { Request, Response } from 'express';
 import * as _ from 'lodash';
-
-import { fetchSite, getHubApiUrl, getPortalApiUrl, hubApiRequest, IHubRequestOptions, RemoteServerError } from '@esri/hub-common';
-import { IContentSearchRequest, SortDirection } from '@esri/hub-search';
-
+import { RemoteServerError } from '@esri/hub-common';
```

Comment:
I think this import is no longer require?

Reply:
Unused import has been removed.